doc/spec/tags.md

```diff
@@ -0,0 +1,149 @@
+# Tags and Signatures
+
+## Introduction
+
+Classically, the role of tagging has been less formal in the docker eco-
+system. Effectively, the concept of "tagging" meant that one assigned a name
+to an image id.
+
+Tags in docker are missing several features that one might expect when coming
+from other systems. In git, one can tag any commit and then sign that tag,
+providing a name to a given revision and the ability to verify that the name
+was assigned by a trusted party. We'd like bring this 
```

Comment:
I was trying to get this out as a preview. I'll give this a full once over today.